packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -449,10 +446,8 @@ export function requestUpdateLane(
     currentEventWipLanes = workInProgressRootIncludedLanes;
   }
 
-  if (suspenseConfig !== null) {
-    // Use the size of the timeout as a heuristic to prioritize shorter
-    // transitions over longer ones.
-    // TODO: This will coerce numbers larger than 31 bits to 0.
+  const isTransition = requestCurrentTransition() !== NoTransition;
```

Comment:
Nit: reading this makes me think "is what a transition" but something like `isInsideTransition` or `isTransitionUpdate` would help understand that we're tracking what type of update we're in.